Makefile

```diff
@@ -65,13 +65,13 @@ localtest:
 	make localunittest localintegration localrootlessintegration
 
 unittest: runcimage
-	docker run -e TESTFLAGS -t --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_IMAGE) make localunittest
```

Comment:
Should `/lib/modules` be read-only? I'm a little concerned about making it read-write (in case something breaks and we overwrite some critical modules).

Reply:
You are right. Will fix.